specification/datadog/resource-manager/Microsoft.Datadog/stable/2022-08-01/datadog.json

```diff
@@ -0,0 +1,2013 @@
+        "modelAsString": true,
+        "name": "ProvisioningState"
+      },
+      "readOnly": true
+    },
+    "MonitoringStatus": {
+      "description": "Flag specifying if the resource monitoring is enabled or disabled.",
+      "enum": [
+        "Enabled",
+        "Disabled"
+      ],
+      "default": "Enabled",
+      "type": "string",
+      "x-ms-enum": {
+        "modelAsString": true,
+        "name": "MonitoringStatus"
+      }
+    },
+    "MarketplaceSubscriptionStatus": {
+      "description": "Flag specifying the Marketplace Subscription Status of the resource. If payment is not made in time, the resource will go in Suspended state.",
+      "enum": [
+        "Provisioning",
+        "Active",
+        "Suspended",
+        "Unsubscribed"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "modelAsString": true,
+        "name": "MarketplaceSubscriptionStatus"
+      },
+      "readOnly": true
+    },
+    "DatadogOrganizationProperties": {
+      "description": "Datadog organization properties",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Name of the Datadog organization.",
+          "type": "string",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "id": {
+          "description": "Id of the Datadog organization.",
+          "type": "string",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "linkingAuthCode": {
+          "description": "The auth code used to linking to an existing datadog organization.",
+          "type": "string",
+          "x-ms-mutability": [
+            "create"
+          ],
+          "x-ms-secret": true
+        },
+        "linkingClientId": {
+          "description": "The client_id from an existing in exchange for an auth token to link organization.",
+          "type": "string",
+          "x-ms-mutability": [
+            "create"
+          ],
+          "x-ms-secret": true
+        },
+        "redirectUri": {
+          "description": "The redirect uri for linking.",
+          "type": "string",
+          "x-ms-mutability": [
+            "create"
+          ]
+        },
+        "apiKey": {
+          "description": "Api key associated to the Datadog organization.",
+          "type": "string",
+          "x-ms-mutability": [
+            "create"
+          ],
+          "x-ms-secret": true
+        },
+        "applicationKey": {
+          "description": "Application key associated to the Datadog organization.",
+          "type": "string",
+          "x-ms-mutability": [
+            "create"
+          ],
+          "x-ms-secret": true
+        },
+        "enterpriseAppId": {
+          "description": "The Id of the Enterprise App used for Single sign on.",
+          "type": "string",
+          "x-ms-mutability": [
+            "create"
+          ]
+        },
+        "cspm": {
```

Comment:
>cspm [](http://example.com/codeflow?start=9&length=4)

Consider to use descriptive name instead of short names for better readability

Reply:
Point taken. Since this will be a breaking change, we'll raise a separate PR just to accommodate all the changes mentioned by you. Currently, the purpose of the PR is to accommodate multi-subscription changes.